docs/prow/prow-installation-on-forks.md

```diff
@@ -0,0 +1,191 @@
+    - Kubernetes Engine Cluster Admin (`roles/container.clusterAdmin`)
+    - Service Account User (`roles/iam.serviceAccountUser`)
+- **sa-vm-kyma-integration** with roles that allow the account to provision virtual machines:
+    - Compute Instance Admin (beta) (`roles/compute.instanceAdmin`)
+    - Compute OS Admin Login (`roles/compute.osAdminLogin`)
+    - Service Account User (`roles/iam.serviceAccountUser`)
+- **sa-gcs-plank** with the role that allows the account to store objects in a Bucket:
+    - Storage Object Admin (`roles/storage.objectAdmin`)
+- **sa-gcr-push** with the role that allows the account to push images to Google Container Repository:
+    - Storage Admin `roles/storage.admin`
+
+See an example of variables you need to export for each account:
+
+```
+export SA_NAME=sa-gcs-plank
+export SA_DISPLAY_NAME=sa-gcs-plank
+export SECRET_FILE=sa-gcs-plank
+export ROLE=roles/storage.objectAdmin
+
+```
+
+## Install Prow
+
+Follow these steps:
+
+1. Export these environment variables:
+
+- **BUCKET_NAME** is a GCS bucket in the Google Cloud project that is used to store Prow Secrets.
+- **KEYRING_NAME** is the KMS key ring.
+- **ENCRYPTION_KEY_NAME** is the key name in the key ring that is used for data encryption.
+
+The account files are encrypted with the **ENCRYPTION_KEY_NAME** key from **KEYRING_NAME** and are stored in **BUCKET_NAME**.
+
+2. Go to the `development` folder and run the following script to start the installation process:
+
+```bash
+./install-prow.sh
+```
+
+> **NOTE:** The scripts prompts you to enter your OAuth2 token.
+
+This script performs the following steps to install Prow:
+
+- Deploy the NGINX Ingress Controller.
+- Create a ClusterRoleBinding.
+- Create a HMAC token to be used for GitHub Webhooks.
+- Create secrets for HMAC and OAuth2 to be used by Prow.
+- Deploy Prow components using the `starter.yaml` file from the `prow/cluster` directory.
+- Add annotations for the Prow Ingress to make it work with the NGINX Ingress Controller.
+
+## Verify the Installation
+
+Verify if Prow installed successfully.
+
+1. Check if all Pods are up and running:
+
+   ```
+   kubectl get pods
+   ```
+
+2. Check if the Deck is accessible from outside of the cluster:
+
+   ```
+   kubectl get ingress ing
+   ```
+
+   Copy the address of the ingress `ing` and open it in a browser to display the Prow status on the dashboard.
+
+## Configure the webhook
+
+After Prow installs successfully, you need to:
+
+1. [Configure a webhook](https://support.hockeyapp.net/kb/third-party-bug-trackers-services-and-webhooks/how-to-set-up-a-webhook-in-github) to enable sending Events from a GitHub repository to Prow.
+2. Add the bot account to [collaborators](https://help.github.com/articles/adding-outside-collaborators-to-repositories-in-your-organization/) on your forked repository and set it with push access rights. The bot account needs to accept your invitation.
```

Comment:
Hm... similar point is in `## Create a bot account` section.

Reply:
Yup, I fixed that. Now it is mentioned only once ;)